src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -483,6 +488,17 @@ export class ExposureNotificationService {
     return true;
   };
 
+  public getExposureDetectedAt(): number | undefined {
+    const exposureStatus = this.exposureStatus.get();
+    let timeStamp;
+
+    if (exposureStatus.type === ExposureStatusType.Exposed && exposureStatus.exposureDetectedAt) {
```

Comment:
It's not necessary to check for `exposureStatus.type === ExposureStatusType.Exposed`.

Reply:
TypeScript thing

![Screen Shot 2020-11-17 at 9 44 11 AM](https://user-images.githubusercontent.com/62242/99404338-7f9d2d00-28b9-11eb-974d-4e24ea8f8b01.png)

Note that block of code has been updated but same comment.